apps/vc-api/docs/example-exchange-definitions/self-signed-vc-exchange.md

```diff
@@ -0,0 +1,112 @@
+                                        "eliaGroupContext":{
+                                            "type":"object",
+                                            "properties":{
+                                              "elia":{
+                                                  "const":"https://www.eliagroup.eu/ld-context-2022#"
+                                              },
+                                              "consent":{
+                                                  "const":"elia:consent"
+                                              }
+                                            },
+                                            "additionalProperties":false,
+                                            "required":[
+                                              "elia",
+                                              "consent"
+                                            ]
+                                        }
+                                      }
+                                  }
+                                },
+                                {
+                                  "path":["$.credentialSubject"],
+                                  "filter":{
+                                      "type":"object",
+                                      "properties":{
+                                        "consent":{
+                                            "const":"I consent to such and such"
+                                        }
+                                      },
+                                      "additionalProperties":true
+                                  }
+                                },
+                                {
+                                  "path":["$.type"],
+                                  "filter":{
+                                      "type":"array",
+                                      "items":[
+                                          {
+                                              "const":"VerifiableCredential"
+                                          }
+                                      ]
+                                  }
+                                }
+                              ]
+                          }
+                        }
+                    ]
+                }
+            }
+         ]
+      }
+    ],
+    "interactServices": [
+      {
+        "type": "UnmediatedHttpPresentationService2021"
+      }
+    ],
+    "callback": [
+      {
+        "url": "FILL YOUR CALLBACK URL, for example 'https://webhook.site/efb19fb8-2579-4e1b-8614-d5a03edaaa7a'"
+      }
+    ],
+    "isOneTime":true
+}
+```
+
+2. For obtaining a credential based on presentation of self-signed Verifiable Credential, the exchange-definition mentioned above could be used with an `interactServices` of type `MediatedHttpPresentationService2021`. The credential could be issued to _DID_ which created the Verifiable Presentation.
```

Comment:
Is this text still needed?

Reply:
As we have `one-presentation-one-issuance` example already

Reply:
We can remove it (we have another example for this purpose).